### DISCUSSION TITLE
Issue with hdpi (4k) monitor

### DISCUSSION THREAD

**Discussion by Franco-64** [Stellarium/stellarium#3572] [Q&A]
Hi all, first post here.

I'm using Stellarium on my laptop with Win11. No problem when I use the default laptop monitor, but I'm trying to setup in my secondary display, that is a 4K (3840x2160). After I changed some line in [video] section of config.ini, Stellarium opens correctly in the secondary 4K display, at full screen. But the planetary image occupies only a section of the screen (see attached image). It seems also that the selection of sky objects in misaligned with what I see in monitor (I can click and have some results in the black area), as the app sees "internally" as the visualization was in full screen.

This is my video section of config.ini:

[video]
dithering_mode                         = color888
fullscreen                             = false
maximum_fps                            = 10000
minimum_fps                            = 18
screen_h                               = 2160
screen_number                          = 1
screen_w                               = 3840
screen_x                               = 0
screen_y                               = 0
tm_display_adaptation_luminance        = 50
viewport_effect                        = none

How can I resolve?

<img width="1919" alt="stellarium" src="https://github.com/Stellarium/stellarium/assets/154953541/3d45f9d0-6d30-4f87-b3ca-0565c4c923ab">

**Comment by alex-w** [comment]
Twice pressing F11 does help?

**Reply by Franco-64** [reply]
No...

**Reply by alex-w** [reply]
Probably same issue as #2490 - please check the report

**Reply by alex-w** [reply]
Please fill a bug report and attach the log from Stellarium

**Reply by Franco-64** [reply]
I've seen #2490, my OpenGL version is 4.6 and is enabled. Should I proceed with a bug report?

**Comment by alex-w** [comment]
> I've seen #2490, my OpenGL version is 4.6 and is enabled. Should I proceed with a bug report?

Yes, because this is a separate issue apparently

**Reply by Franco-64** [reply]
Ok done!
